### PR TITLE
Changing domain in hyperlinks

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Replacing domain: openvinotoolkit.org to openvino.ai in hyperlinks. 

Redirection from links such as:

https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html

results in "404 - Page not found".

My guess is that rule in .htaccess file changes the url text to lower case.

### CODE REVIEW

**Comment by ryanloney** [comment]
@sgolebiewski-intel this PR is not passing CI tests. Do you have some outputs in the notebooks that are not cleared out? Or something else causing the failure?

File: notebooks/102-pytorch-onnx-to-openvino/102-pytorch-onnx-to-openvino.ipynb
Comment (paularamo) [inline notebooks/102-pytorch-onnx-to-openvino/102-pytorch-onnx-to-openvino.ipynb:553]: Page not found

File: notebooks/205-vision-background-removal/205-vision-background-removal.ipynb
Comment (paularamo) [inline notebooks/205-vision-background-removal/205-vision-background-removal.ipynb:502]: Page not found

File: notebooks/206-vision-paddlegan-anime/206-vision-paddlegan-anime.ipynb
Comment (paularamo) [inline notebooks/206-vision-paddlegan-anime/206-vision-paddlegan-anime.ipynb:593]: Page not found

File: notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino-pot.ipynb
Comment (paularamo) [inline notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino-pot.ipynb:503]: Page not found

File: notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino-pot.ipynb
Comment (paularamo) [inline notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino-pot.ipynb:619]: Page not found

File: notebooks/301-tensorflow-training-openvino/README.md
Comment (paularamo) [inline notebooks/301-tensorflow-training-openvino/README.md:10]: Page not found

**Review by paularamo — CHANGES_REQUESTED** [review]

File: notebooks/301-tensorflow-training-openvino/README.md
Comment (paularamo) [reply notebooks/301-tensorflow-training-openvino/README.md:10]: @sgolebiewski-intel Thanks for the update about hyperlinks. Some URLs are not working. Please take a look at my comments. BR.

**Comment by paularamo** [comment]
@sgolebiewski-intel Thanks for the update about hyperlinks. Some URLs are not working. Please take a look at my comments. BR.

**Comment by sgolebiewski-intel** [comment]
> @sgolebiewski-intel this PR is not passing CI tests. Do you have some outputs in the notebooks that are not cleared out? Or something else causing the failure?

Found out that change of the domain in urls

**Comment by sgolebiewski-intel** [comment]
> @sgolebiewski-intel Thanks for the update about hyperlinks. Some URLs are not working. Please take a look at my comments. BR.

Yes, thank you.  I'm getting into it. :)

File: notebooks/301-tensorflow-training-openvino/README.md
Comment (sgolebiewski-intel) [reply notebooks/301-tensorflow-training-openvino/README.md]: Again, thank you for pointing that out. I have corrected them, and they seem to be working now.

**Review by paularamo — APPROVED** [review]
Thanks, @sgolebiewski-intel for changing the URLs. We should have to see the @ryanloney approval before merging.

**Comment by sgolebiewski-intel** [comment]
> @sgolebiewski-intel this PR is not passing CI tests. Do you have some outputs in the notebooks that are not cleared out? Or something else causing the failure?

Turned out, that , after the change, some of the hyperlinks were still broken. Managed to correct them.

**Review by paularamo — APPROVED** [review]

File: notebooks/301-tensorflow-training-openvino/README.md
Comment (sgolebiewski-intel) [reply notebooks/301-tensorflow-training-openvino/README.md]: Hi there, it's seems the build fails because of sth else. I've tracked the logs. If I'm not mistaken, the culprit is:
test_211-speech-to-text.ipynb

**Review by ryanloney — CHANGES_REQUESTED** [review]
@sgolebiewski-intel please see suggested edits. Need to fix a few URLs. 

Also, I emailed the documentation team to see why we don't have 'latest' version of ONNX Support page. (and only 2021.4)

**Pull request merged**
